ksqldb-engine/src/main/java/io/confluent/ksql/planner/LogicalPlanner.java

```diff
@@ -405,7 +407,7 @@ private PreJoinRepartitionNode buildInternalRepartitionNode(
         ExpressionTreeRewriter.rewriteWith(plugin, joinExpression);
 
     final LogicalSchema schema =
-        buildRepartitionedSchema(source, rewrittenPartitionBy);
+        buildRepartitionedSchema(source, Collections.singletonList(rewrittenPartitionBy));
```

Comment:
nit: since we already import `ImmutableList` let's just use `ImmutableList.of` here

Reply:
You mean we import ImmutableList in this file or in the module? I don't see it in the file but I can add it.

Reply:
oh huh, I must've been looking at a separate file. - in `PreJoinRepartitionNode` for example it was already there before this PR. It's nice to just standardize on one (and most of hte code uses `ImmutableList#of`, though there isn't really any good reason to use that over `Collections.singletonList` except for mabye that `ImmutableList` has more general usages)